buildspecs/python2.7.yml

```diff
@@ -1,9 +1,11 @@
-version: 0.1
+version: 0.2
 phases:
   pre_build:
     commands:
-      - pip install -r ./requirements.txt
+      - pip install -r ./requirements.txt -t /tmp/vendored
       - rm -rf .git/
+      - cp -r . /tmp/vendored
```

Comment:
What is the benefit of copying everything to this other directory? Would an alternative be to set the `-t` flag on `pip install` to the `${pwd}`?